contracts/Tokens/VTokens/VBNB.sol

```diff
@@ -36,12 +36,22 @@ contract VBNB is VToken {
         admin = admin_;
     }
 
+    /**
+     * @notice Send BNB to VBNB to mint
+     */
+    function() external payable {
+        (uint err, ) = mintInternal(msg.value);
+        requireNoError(err, "mint failed");
+    }
+
     /*** User Interface ***/
 
     /**
      * @notice Sender supplies assets into the market and receives vTokens in exchange
      * @dev Reverts upon any failure
      */
+    // @custom:events Emits Transfer event
```

Comment:
We could avoid mixing styles in the same block. I would add the `custom:events` tags into the `/** ... */` block

Reply:
I'm sorry, now I read your comment in Slack. I suppose you don't want to add these comments in the final doc, right? BTW, why?